zapd/models.py

```diff
@@ -315,11 +315,45 @@ def _format_total_column(view, context, model, name):
             '''.format(payments_url=payments_url, total=total, total_claimed=total_claimed)
         return Markup(html)
 
+    def _format_total_column_text_total(view, context, model, name):
+        if model.status == model.STATE_DECLINED:
+            return Markup('-')
+        total = 0
+        total_claimed = 0
+        for payment in model.payments:
+            total += payment.amount
+            if payment.status == payment.STATE_SENT_FUNDS:
+                total_claimed += payment.amount
+        total = str(total / 100)
+        total_claimed = str(total_claimed / 100)
+        payments_url = url_for('.payments_view', proposal_id=model.id)
+        if total_claimed == total:
+            total=total
+        return Markup(total)
+
+    def _format_total_column_text_totalclaimed(view, context, model, name):
+        if model.status == model.STATE_DECLINED:
+            return Markup('-')
+        total = 0
+        total_claimed = 0
+        for payment in model.payments:
+            total += payment.amount
+            if payment.status == payment.STATE_SENT_FUNDS:
+                total_claimed += payment.amount
+        total = str(total / 100)
+        total_claimed = str(total_claimed / 100)
+        payments_url = url_for('.payments_view', proposal_id=model.id)
+        if total_claimed != total:
+            total_claimed=total_claimed
```

Comment:
this does not actually do anything ??

Reply:
suppose to compare total_claimed against total... but i could change it so its just:
    def _format_total_column_text_totalclaimed(view, context, model, name):
        if model.status == model.STATE_DECLINED:
            return Markup('-')
        total = 0
        total_claimed = 0
        for payment in model.payments:
            total += payment.amount
            if payment.status == payment.STATE_SENT_FUNDS:
                total_claimed += payment.amount
        total_claimed = total_claimed / 100
        payments_url = url_for('.payments_view', proposal_id=model.id)
        return Markup(total_claimed)